apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -133,6 +170,12 @@ class PageTemplateModal extends Component {
 		}
 	}
 
+	componentWillUnmount() {
+		if ( this.props.isPostTitleBlockRegistered ) {
+			unregisterBlockType( 'a8c/spt-template-post-title' );
```

Comment:
Interesting that we don't couple the block _registration_ to a lifecycle of this component but we do hook the unregister to one?